src/main/java/org/marek/model/User.java

```diff
@@ -0,0 +1,18 @@
+package org.marek.model;
+
+import javax.persistence.Entity;
+
+@Entity
+
+public class User {
+    private Integer id;
+    private String name;
+    private String surname;
+    private Integer phoneNumber;
+    private String mail;
+    private String password;
+    private Integer birthday;
+    private Integer createDate;
+    private Integer editDate;
+    private Integer deleteDate;
```

Comment:
Budeme používat jméno `timeDeleted`, tohle vyznívá krkolomně. `@NotNull`. A nebude to `Integer`, ale `LocalDateTime`.